code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -335,3 +335,55 @@
 			message = replacetextEx(message,regex(capitalize(key),"g"), "[capitalize(value)]")
 			message = replacetextEx(message,regex(key,"g"), "[value]")
 	speech_args[SPEECH_MESSAGE] = trim(message)
+	
+/obj/item/clothing/mask/translator
+	name = "galactic translator"
+	desc = "Allows you to speak and understand galactic common and uncommon."
+	icon_state = "tonguerobot"
+	item_state = "sechailer"
+	//clothing_flags = BLOCK_GAS_SMOKE_EFFECT | MASKINTERNALS
+	flags_inv = HIDEFACIALHAIR|HIDEFACE
+	w_class = WEIGHT_CLASS_SMALL
+	flags_cover = MASKCOVERSMOUTH
+	visor_flags_cover = MASKCOVERSMOUTH
+	var/list/languages = list(/datum/language/common, /datum/language/uncommon)
+
+/obj/item/clothing/mask/translator/equipped(mob/user, slot)
+	if(!ishuman(user))
+		return
+	if(slot == SLOT_HEAD)
+	for(var/datum/language/l in languages)
```

Comment:
The fact that this isn't even inside the if statement leads me to think this is untested?